index.js

```diff
@@ -240,7 +241,7 @@ async function terraform(terraformDirectory, args) {
 
   core.startGroup('Run terraform apply');
   if (terraformDoApply === true) {
-    const tfa = await terraform(terraformDirectory, ['apply', `-lock=${terraformLock}`, `-parallelism=${terraformParallelism}`, '-auto-approve'].concat(terraformTargets).concat('terraform.plan'));
+    const tfa = await terraform(terraformDirectory, ['apply', `-lock=${terraformLock}`, `-parallelism=${terraformParallelism}`, `refresh=${terraformRefresh}`, '-auto-approve'].concat(terraformTargets).concat('terraform.plan'));
```

Comment:
Since this uses the plan created in the previous step, refresh might not be needed?

Reply:
And had forgot the dashes :facepalm: